tests/perf/benchmark.py

```diff
@@ -132,13 +133,17 @@ def run(
         model: Model,
         dataset: Dataset,
         criteria: list[Criterion],
+        perf_dir_to_load: str | None = None,
```

Comment:
`Path` is more preferable as other paths are passed as `Path` rather than `str` in this whole module.

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3556/commits/3d6a54e29e75f9e7fc0c36d545d4c25e90881575